libraries/dynamic-ui-react/stories/hooks/useToast.mdx

```diff
@@ -26,8 +27,8 @@ a `DToast` to generate toasts. For more detailed information, refer to the offic
 
 <Unstyled>
   <DAlert theme="warning">
-    toast appears in all containers because of the way react-hot-toast is programmed, however,
-    if you want to see the correct toast make it show up from the correct container.
+    <span>Toast appears in all containers because of the way react-hot-toast is programmed, however,
+    if you want to see the correct toast make it show up from the correct container.</span>
```

Comment:
is a tag

```suggestion
    <span>
      Toast appears in all containers because of the way react-hot-toast is programmed, however,
      if you want to see the correct toast make it show up from the correct container.
     </span>
```